package.json

```diff
@@ -17,15 +17,18 @@
   },
   "dependencies": {
     "brace": "^0.8.0",
+    "classnames": "^2.2.5",
```

Comment:
classnames used by react-flexbox-grid, I tried to omit this dep from packages.json but Travis-CI build result a failed build -> https://travis-ci.org/yasp-dota/ui/builds/149625615